README.md

```diff
@@ -1165,7 +337,7 @@ The API caller uses the transfer host to construct the appropriate transfer clie
 using (ITransferClient client = host.CreateClient(new AsperaClientConfiguration()))
 { }
 
-// I need a file share client.
+// I need a Fileshare client.

```

Comment:
Sometimes it's spelled "file share", sometimes "Fileshare". Azure Portal uses "file share".

Reply:
Done